decidim-assemblies/app/controllers/decidim/assemblies/assemblies_controller.rb

```diff
@@ -77,7 +77,7 @@ def promoted_assemblies
       end
 
       def parent_assemblies
-        search.results.parent_assemblies.order(promoted: :desc)
+        search.results.parent_assemblies.order(weight: :asc)
```

Comment:
This should fallback to the promoted, IMO

Reply:
Do you propose this?

`search.results.parent_assemblies.order(promoted: :desc, weight: :asc)`

Reply:
This would work, yes! Or the other way around, `order(weight: :asc, promoted: :desc)`. Not sure what were the requirements or what's @decidim/product thoughts on this, you might need to ask.

You might need to modify code from the previous PR to keep the same feature, sorry about that!

Reply:
I think product gave the OK to first order them by weight and then as they were normally ordered if they have no weight.

What do you mean about modify code from the previous PR?

Reply:
> What do you mean about modify code from the previous PR?

I mean that probably the same issue happens with the Participatory Processes!

Reply:
Participatory processes are different because in this section we show participatory processes and participatory processes group. 
These processes groups have no weight so it could not be applied.